src/Component/RegistrationForm.js

```diff
@@ -0,0 +1,278 @@
+import React, { Component } from 'react'
+import '../Stylesheet/RegistrationForm.css'
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props)
+    this.isInputsValid = true
+    this.state = {
+      loginId: {
```

Comment:
All of the states look same...write some dynamic code. Add fields in constant and use constant to create the state.

Reply:
Created a method and it will return the value and error object.